src/services/highcharts-factory/highcharts-factory.svc.coffee

```diff
@@ -87,16 +82,32 @@ angular
             y: -5
         }]
 
-    thresholdsMarkers: (options = {})->
-      return {} if _.isEmpty(options.thresholds)
-      lines = []
+    addThresholds: (options = @options)->
```

Comment:
The plotLine doesn't seem to have a Zones API :\ Ok yes I can add an arg for full-length threshold